### ISSUE TITLE
Ionic app missing webpack

### ISSUE THREAD

**Issue by wstidolph** [nstudio/xplat#264]
Following **Getting Started** for Ionic app results in cannot find module 'webpack'

1. fresh 'empty' workspace named 'xptest'
2. install xplat (12.0.2)
2. generate new ionic app 'sample'
3. `nx run ionic-sample:serve`
```
Cannot find module 'webpack'
Require stack:
- /home/wstidolph/dev/xptest/node_modules/@angular-devkit/build-webpack/src/webpack/index.js
- /home/wstidolph/dev/xptest/node_modules/@angular-devkit/build-webpack/src/index.js
- /home/wstidolph/dev/xptest/node_modules/@angular-devkit/build-angular/src/dev-server/index.js
- /home/wstidolph/dev/xptest/node_modules/@angular-devkit/architect/node/node-modules-architect-host.js
- /home/wstidolph/dev/xptest/node_modules/@angular-devkit/architect/node/index.js
- /home/wstidolph/dev/xptest/node_modules/@nrwl/tao/src/commands/ngcli-adapter.js
- /home/wstidolph/dev/xptest/node_modules/@nrwl/tao/src/commands/run.js
- /home/wstidolph/dev/xptest/node_modules/@nrwl/tao/index.js
- /home/wstidolph/dev/xptest/node_modules/@nrwl/cli/lib/run-cli.js

———————————————————————————————————————————————

>  NX   ERROR  Running target "ionic-sample:serve" failed

  Failed tasks:
  
  - ionic-sample:serve
```

Using Win 10 / WLS 2 / Ubuntu 20.10 / node v15.2.1 / npm 7.6.2

**Comment by wstidolph** [comment]
Oops - I didn't run `nx generate @nstudio/xplat:init --platforms web,ionic` ... maybe that explains this?

**Comment by NathanWalker** [comment]
@wstidolph possible - most of the app generators will implicitly generate the supporting architecture if needed so something may have occurred there. We can check.

**Comment by wstidolph** [comment]
Well, that didn't fix it; should I move the error stack below to being a new issue or is it legit related to the 'missing webpack' issue root cause?
````
nx generate @nstudio/xplat:init --platforms web,ionic
✔ Do you prefer to use a frontend framework? · angular
✔ Please input a 2-3 character prefix to use with xplat generators: · ss
Note to VS Code users: no development mode set. xplat could not find any VS Code settings in the standard location: /var/local/Code/User/settings.json It's possible you don't have a user settings.json yet. If so, open VS Code User settings and save any kind of setting to have it created.
✔ Packages installed successfully.
✔ Packages installed successfully.
Note to VS Code users: no development mode set. xplat could not find any VS Code settings in the standard location: /var/local/Code/User/settings.json It's possible you don't have a user settings.json yet. If so, open VS Code User settings and save any kind of setting to have it created.
⠼ Installing packages (npm)...npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR! 
npm ERR! While resolving: xptest@0.0.0
npm ERR! Found: jasmine-core@3.6.0
npm ERR! node_modules/jasmine-core
npm ERR!   dev jasmine-core@"~3.6.0" from the root project
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer jasmine-core@">=3.7.1" from karma-jasmine-html-reporter@1.6.0
npm ERR! node_modules/karma-jasmine-html-reporter
npm ERR!   dev karma-jasmine-html-reporter@"^1.5.2" from the root project
npm ERR! 
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR! 
npm ERR! See /home/wstidolph/.npm/eresolve-report.txt for a full report.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/wstidolph/.npm/_logs/2021-05-07T17_43_15_789Z-debug.log
✖ Package install failed, see above.
UnsuccessfulWorkflowExecution [Error]: Workflow did not execute successfully.
    at ChildProcess.<anonymous> (/home/wstidolph/dev/xptest/node_modules/@angular-devkit/schematics/tasks/package-manager/executor.js:109:31)
    at ChildProcess.emit (node:events:329:20)
    at maybeClose (node:internal/child_process:1055:16)
    at Process.ChildProcess._handle.onexit (node:internal/child_process:288:5)
The generator workflow failed. See above.
````
[2021-05-07T17_43_15_789Z-debug.log](https://github.com/nstudio/xplat/files/6444658/2021-05-07T17_43_15_789Z-debug.log)
[eresolve-report.txt](https://github.com/nstudio/xplat/files/6444662/eresolve-report.txt)

**Comment by NathanWalker** [comment]
Fixed in 12.4.1
